test/mock_compatibility.js

```diff
@@ -0,0 +1,87 @@
+// For the convenience of unit testing, add these to the global namespace
+var _      = require('underscore');
+var expect = require('chai').expect;
+
+// Unit testing is done against the debug version of the library as it has
+// additional conformance checks that are optimized out of the production
+// library.  Again, globals are used purely for convenience.
+var opentracing = require('../debug.js');
+
+module.exports = function mockCompatibilityChecks() {
+    describe('Mock Opentracing Implementation', function() {
+        describe('Constants', function() {
+            var constStrings = [
+                'FORMAT_TEXT_MAP',
+                'FORMAT_BINARY',
+                'FORMAT_HTTP_HEADERS',
+                'REFERENCE_CHILD_OF',
+                'REFERENCE_FOLLOWS_FROM',
+            ];
+            _.each(constStrings, function(name) {
+                it(name + ' should be a constant string', function() {
+                    expect(opentracing[name]).to.be.a('string');
+                });
+            });
+        });
+
+        describe('Standalone functions', function() {
+            var funcs = [
+                'childOf',
+                'followsFrom',
+                'initGlobalTracer',
+                'globalTracer',
```

Comment:
I think this should be just a standalone test. It's not testing "compatibility", it's testing public opentracing API